docs/index.rst

```diff
@@ -6,8 +6,12 @@
 KA Lite Documentation
 ===================================
 
-Welcome to the KA Lite Documentation page! Here, you will find all the information needed to set up the KA Lite software. Additionally, there's information on how to manage KA Lite and how to perform role-specific tasks. If you have any questions that are not addressed by the documentation and you have access to the Internet, please be sure to check our `FAQ`_ or `Forums`_!    
-
+Welcome to the KA Lite Documentation page! Here, you will find all the 
+information needed to set up the KA Lite software. Additionally, there's 
```

Comment:
I was editing in nano, and I couldn't read the line, so I just CTRL+J'ed it. Wrapping at 80 chars matters IMO.

It also helps you not creating too long paragraphs -- equally important as the sentence length ;)